BuildingSync.xsd

```diff
@@ -405,6 +405,19 @@
           </xs:complexType>
         </xs:element>
       </xs:sequence>
+      <xs:attribute name="version" use="required">
+        <xs:simpleType>
+          <xs:restriction base="xs:string">
```

Comment:
Please sort the enumeration values.

Reply:
They are sorted by official release date (reverse order).  How would you like to see it?

Reply:
That makes sense. Please add an annotation element with documentation for the list ordering, rationale, etc., and a link to the "legacy" version.

Reply:
That works